src/goes/goes.py

```diff
@@ -240,41 +132,29 @@ def _string_to_abisectortype(string):
         if 'M2' in string:
             return ABISectorType.MESOSCALE_REGION_2
 
-    def _filter_data_array_by_yaw_flip_flag(self, data_array):
-        """
-        Returns data_array after filtering by the yaw_flip_flag.
-        data_array - the data array to filter
-        """
-        if not self._yaw_flip_flag:
-            return data_array[::-1]
-        else:
-            return data_array
-
     def _create_obsvalue_rf_data_array(self):
         """
         Creates a local data array variable containing the calculated obsvalue reflectance factor data
         after fill value filtering by the DQF flags.
         """
-        temp_data_array = self._rad_data_array * self._kappa0
-        self._obsvalue_rf_data_array = np.where(self._dqf_data_array == -999, -999, temp_data_array)
+        self._obsvalue_rf_data_array = self._rad_data_array * self._kappa0
 
     def _create_obsvalue_bt_data_array(self):
         """
         Creates a local data array variable containing the calculated obsvalue brightness temperature data
         after fill value filtering by the DQF flags.
         """
         log_comp = np.log((self._planck_fk1 / self._rad_data_array) + 1)
-        temp_data_array = ((self._planck_fk2 / log_comp) - self._planck_bc1) / self._planck_bc2
-        self._obsvalue_bt_data_array = np.where(self._dqf_data_array == -999, -999, temp_data_array)
+        self._obsvalue_bt_data_array = ((self._planck_fk2 / log_comp) - self._planck_bc1) / self._planck_bc2
```

Comment:
Comment: We should probably compare this TB computation with CRTM's internal function for computing TBs, to ensure consistency.    Can you create a comment here identifying the source of this equation/relationship?

Reply:
The sources for all formulations used in each python class are listed in the header of each class. Yes. I would like to compare this to the TB (assuming that means Temperature Brightness) to the one used in CRTM.